crowdin/src/main/java/com/crowdin/platform/data/remote/StringDataRemoteRepository.kt

```diff
@@ -68,6 +70,11 @@ internal class StringDataRemoteRepository(
                     body
                 )
             }
+            code == HttpURLConnection.HTTP_FORBIDDEN -> {
+                val errorMessage = "Resource file, for locale - ${Locale.getDefault()}, not found"
```

Comment:
Can we do this?
`Translation file ${filePath} for locale ${Locale.getDefault()} not found in the distribution`

Reply:
Updated.